proxysql_tools/cli.py

```diff
@@ -199,6 +199,24 @@ def set_sync(cfg, ip_address, port):
         LOG.error(err)
 
 
+@server.command()
+@click.argument('ip_address', required=True)
+@click.argument('port', required=False, type=int, default=3306)
+@PASS_CFG
+def set_online(cfg, ip_address, port):
+    """Set server backend status online"""
+    server_set_admin_status(cfg, ip_address, port, BackendStatus.online)
```

Comment:
Passing cfg down is a bad design practice. Let's discuss offline how we can refactor it